src/pacing.py

```diff
@@ -55,15 +55,13 @@ class Binning(PacingFunction):
     def __init__(self, args):
         super(Binning, self).__init__()
         self.bin_variable = args.bin_variable
+        self.descending = args.descending  
         self.bin_method = args.bin_method
         self.n_bins = args.n_bins
         self.bin_validation_metric = args.bin_validation_metric
 
+    #def __call__(self, df, descending=False):
     def __call__(self, df):
-        df = df.sort_values(by=[self.bin_variable])
```

Comment:
I see you've removed "df = df.sort_values(by=[self.bin_variable])" in "src/pacing.py", why is that? Isn't this necessary?

Reply:
I don't find it necessary for CL learning purposes: pandas' cut/qcut functions do the work. When using binning as a pacing function, we are interested in creating bins, each including samples within a range of the binning variable. The order in which samples are sorted in each bin is not relevant: batches will be selected randomly during training.

It will make no harm to sort them, though. If the df is sorted before creating the bins, the resulting manifests for each bin will also be sorted. This can be useful for analysis, or if in the future we want to carry out experiments in which at every step of the CL strategy inputs are given in a sorted manner.